chain/network/src/peer_manager/network_state/mod.rs

```diff
@@ -83,6 +94,8 @@ pub(crate) struct NetworkState {
     runtime: Runtime,
     /// PeerManager config.
     pub config: config::VerifiedConfig,
+    /// Store for read/write access to recent outbound connections.
+    pub store: Mutex<store::Store>,
```

Comment:
Passing a mutex by value doesn't make sense.
You either want to share a value under mutex (then you use Arc<Mutex<...>>) or
you don't and then you just pass a value directly (without mutex)